pkg/scannerv4/mappers/mappers.go

```diff
@@ -405,6 +458,20 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 			// The preferred CVSS metrics will always be stored at the first index.
 			preferredCVSS = metrics[0]
 		}
+
+		if !features.ScannerV4RedHatCVEs.Enabled() {
+			rhelEPSSDetails = rhelVulnsEPSS(vulns, cveEPSS)
```

Comment:
I think this map should be built before the for loop. Otherwise it will run for all vulns in each iteration.